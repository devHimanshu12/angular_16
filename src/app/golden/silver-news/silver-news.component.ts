import { Component } from '@angular/core';

@Component({
  selector: 'app-silver-news',
  templateUrl: './silver-news.component.html',
  styleUrls: ['./silver-news.component.scss']
})
export class SilverNewsComponent {

}
