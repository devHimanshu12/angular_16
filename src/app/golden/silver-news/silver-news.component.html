<p>silver-news works!</p>
